docs_src/design/adr/core/0022-UoM.md

```diff
@@ -0,0 +1,199 @@
+
+- [ANSI X12: EDI standard used mostly in the US](https://ediacademy.com/blog/x12-unit-of-measurement-codes/)
+- [EDIFACT: UN EDI standard used mostly in Europe and Asia](https://unece.org/fileadmin/DAM/cefact/recommendations/rec20/rec20_rev3_Annex2e.pdf)
+
+The [Unified Code for Units of Measure](https://en.wikipedia.org/wiki/Unified_Code_for_Units_of_Measure) provides an alternative list (not a standard) that is used by various organizations like OSGI and the Eclipse Foundation.
+
+While standards exist, use by various open source projects (especially IoT/edge projects) is inconsistent and haphazard.   Groups like oneM2M seem to define their own selection of units in specifications per vertical (home for example) while Kura doesn't even appear to use the UoM JSR (a Java related unit of measure specification for Java applications like Kura).
+
+## Decision
+
+It would be speculative and inappropriate for EdgeX to select a unit of measure standard which is not widely adopted in the industry or choose a static unit of measure list that is incomplete with regard to possible IoT / edge use case needs.  At this time, there does not appear to be a single and unequivocal standard for units of measure that encompasses all EdgeX related use cases (now and in the future). 
+
+Therefore, EdgeX chooses not to select or adopt a unit of measure specification, standard, or code list to apply across the platform.  Instead, EdgeX adopters will be allowed to optionally specify which unit of measure specification, standard, or unit of measure code list they would like used in their instance(s) of EdgeX.
+
+### Specifying the Units of Measure
+
+Units of measure allowed by the instance of EdgeX will be specified in a configuration file (in TOML format).  This can either be specified directly in the core metadata configuration file (but then hard to change when using a containerized or snap form of the application) or in a separate and external configuration TOML file (making it easier for an adopter to provide units of measure even when the application is containerized or snapped up).  
+
+The units of measure in the configuration file can be attributed, optionally, to a specification, document, or other UoM definition source.  The `source` only helps provide the location of documentation about the origins and details of the units specified for the reader, but it will not be used or checked by EdgeX.  An optional default source can be provided at the top level configuration (as shown in the examples below) so that other sources are only need when there specific units used that are not found in the default source.
```

Comment:
```suggestion
The units of measure in the configuration file can be attributed, optionally, to a specification, document, or other UoM definition source.  The `source` only helps provide the location of documentation about the origins and details of the units specified for the reader, but it will not be used or checked by EdgeX.  An optional default source can be provided at the top level configuration (as shown in the examples below) so that other sources are only needed when there are specific units that are not found in the default source.
```

Reply:
fixed